packages/manager/apps/hpc-vmware-managed-vcd/src/components/form/DatacentreOrder.component.tsx

```diff
@@ -111,42 +111,40 @@ export const DatacentreOrder = <T extends OrderType>({
     <React.Suspense>
       <div className="px-10 my-4 flex flex-col">
         <Subtitle>{title}</Subtitle>
-        <Description className="my-6">{subtitle}</Description>
+        <OdsText className="my-6">{subtitle}</OdsText>
         <Datagrid
           columns={columns}
           items={pricedResources}
           totalItems={pricedResources.length}
           contentAlignLeft
         />
         <div className="mt-10">
-          <QuantitySelector
-            quantity={selectedQuantity}
-            setQuantity={setSelectedQuantity}
-            isValid={isValidQuantity}
-            title={t('managed_vcd_vdc_order_quantity_title')}
-            label={t('managed_vcd_vdc_order_quantity_label')}
-            min={minQuantity}
-            max={maxQuantity}
-          />
+          <Subtitle>{t('managed_vcd_vdc_order_quantity_title')}</Subtitle>
+          <div className="flex flex-col items-start">
+            <OdsText class="my-2">
+              {t('managed_vcd_vdc_order_quantity_label')}
+            </OdsText>
+            <OdsQuantity
+              name="order-quantity"
+              min={minQuantity}
+              max={maxQuantity}
+              hasError={!isValidQuantity}
+              value={selectedQuantity}
+              onOdsChange={(e) => setSelectedQuantity(e.detail.value)}
+            />
+          </div>
         </div>
         <div className="flex items-center gap-x-4 mt-10">
-          <OsdsButton
-            size={ODS_BUTTON_SIZE.sm}
-            variant={ODS_BUTTON_VARIANT.ghost}
-            color={ODS_THEME_COLOR_INTENT.primary}
+          <OdsButton
+            label={t('managed_vcd_vdc_order_cancel_cta')}
+            variant="ghost"
             onClick={() => navigate(backLink)}
-          >
-            {t('managed_vcd_vdc_order_cancel_cta')}
-          </OsdsButton>
-          <OsdsButton
-            size={ODS_BUTTON_SIZE.sm}
-            variant={ODS_BUTTON_VARIANT.flat}
-            color={ODS_THEME_COLOR_INTENT.primary}
-            disabled={!isValidQuantity || undefined}
+          />
+          <OdsButton
+            label={t('managed_vcd_vdc_order_confirm_cta')}
+            isDisabled={!isValidQuantity || undefined}
```

Comment:
```suggestion
            isDisabled={!isValidQuantity}
```


It's fixed in ODS 18 ?